lib/rouge/lexers/javascript.rb

```diff
@@ -206,7 +206,7 @@ class JSON < RegexLexer
       desc "JavaScript Object Notation (json.org)"
       tag 'json'
       filenames '*.json'
-      mimetypes 'application/json'
+      mimetypes 'application/json', 'application/vnd.api+json'
```

Comment:
Perhaps `application/(.*)+json` could match?

This way other JSON based hypermedia types will be included:

- `application/json`
- `application/hal+json`
- `application/vnd.siren+json`
- `application/vnd.api+json`